### ISSUE TITLE
Correcting the "will" and "would" future cases

### ISSUE THREAD

**Issue by PTB-OEDA** [openeventdata/UniversalPetrarch#34]
There is an issue with these future verbs that leads to too many "04" cases in P2 and UDP coding.  This needs to be fixed in the PICO dictionary if I have understood the issue correctly.  We need to do this on the EN side, since it will only get much worse when we add more languages like ES and AR.

**Comment by ahalterman** [comment]
Yes, what's the plan for updating PICO in general? We talked a lot about it this spring but it seems to have stalled.

**Comment by ahalterman** [comment]
And related to this, how is the Spanish team/UP handling future tenses? Via the dictionary or is there a more general approach using the Universal Dependencies information (http://universaldependencies.org/u/feat/Tense.html)?
